_starter-kit/component-generator/index.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+var yeoman = require('yeoman-generator');
+var includes = require('lodash.includes');
+var path = require('path');
+var fs = require('fs');
+var chalk = require('chalk');
+var plBase = ('./pattern-lab/source/_patterns');
```

Comment:
Is something missing here? The parenthesis wrapping combined with the previous few lines makes it seem like this line might be missing a `require` call (though I don't believe it is?)

Reply:
Since this is just a path, I don't think needs the require line or any changes.